demos/python_demos/multi_camera_multi_person_tracking/README.md

```diff
@@ -0,0 +1,83 @@
+# Multi Camera Multi Person Python* Demo
+
+This demo demonstrates how to run Multi Camera Multi Person demo using OpenVINO<sup>TM</sup>
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu 16.04
+* Python 3.5.2
+* OpenVINO 2019 R2 with Python API
```

Comment:
I don't think this needs to be mentioned. The demos are installed as part of the OpenVINO toolkit; there is no way for these prerequisites to be unfulfilled.